lib/routes/searches.js

```diff
@@ -0,0 +1,34 @@
+const router = require('express').Router();
+const Search = require('../model/Search');
+
+router.post('/', (request, response) => {
+  const tournament = request.body.tournament;
+  const user = request.body.user;
+
+  const newSearch = new Search({
+    tournament,
+    user,
+  });
+
+  newSearch
+    .save()
+    .then((newSearch) => response.json(newSearch))
+    .catch((err) => response.json('Error:' + err));
```

Comment:
You mix async-await/try-catch and then-catch in your project. 
I recommend selecting one of these solutions.